lib/internal/test_runner/harness.js

```diff
@@ -217,9 +218,24 @@ function runInParentContext(Factory) {
     return PromiseResolve();
   }
 
-  const test = (name, options, fn) => run(name, options, fn);
+  const test = (name, options, fn) => {
+    const overrides = {
+      __proto__: null,
+      loc: getCallerLocation(),
```

Comment:
What is the overhead of calling this?

Reply:
Any way to collect this only for failed tests? ( My intuition is no - it will be wrong like the stack trace, just asking)

Reply:
I don't think there is a way to get the information for failed tests only - at least not accurately. You can get the location of the test function at any point, but that is inaccurate in a few cases. For example:

```js
function myTest() {
  // ...
}

test('my test', myTest);
```